common/include/pcl/point_cloud.h

```diff
@@ -631,6 +637,11 @@ namespace pcl
       void
       inline assign(std::initializer_list<PointT> ilist, index_t new_width)
       {
+        if (new_width == 0) {
+          PCL_WARN("Assignment with new_width equal to 0,"
+                   "setting width to size of the cloud and height to 1");
+          return assign(std::move{ilist));
```

Comment:
Syntax error here (curly bracket)

Reply:
Now this is a problem again :slightly_frowning_face:

Reply:
```suggestion
          return assign(std::move(ilist));
```

Reply:
> Now this is a problem again slightly_frowning_face

Sorry,I am working from 2 different machines, and probably made a git mistake